### PR TITLE
Feature/bump to specific version

### LINKED ISSUE #188: Bump to an exact release version
Example use cases:
- Align on a version across all crates in a workspace
- Bumping major with a pre-release after introducing breaking changes.  Currently you can only bump major or pre-release, not both.

### PR DESCRIPTION
**Pull request by sunng87**
Fixes #188 

This patch added support for bumping version to a fixed one. And the given version has to be:

* A valid semver string
* A version that is greater than current one

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -767,12 +786,8 @@ struct ReleaseOpt {
...
 
     /// Release level: bumping specified version field or remove prerelease extensions by default
-    #[structopt(
-        possible_values(&version::BumpLevel::variants()),
-        case_insensitive(true),
-        default_value = "release"
-    )]
-    level: version::BumpLevel,
+    #[structopt(case_insensitive(true), default_value = "release")]
+    level: String,
```
Comment (epage) [inline src/main.rs]: What about renaming this to `level_or_version`?  iirc that will make the intent clearer in `--help`

File: src/main.rs [hunk 2]
```diff
@@ -767,12 +786,8 @@ struct ReleaseOpt {
     workspace: clap_cargo::Workspace,
 
     /// Release level: bumping specified version field or remove prerelease extensions by default
```
Comment (epage) [inline src/main.rs]: Does the `--help` text (via the doc) need updating?

File: src/main.rs [hunk 3]
```diff
@@ -767,12 +786,8 @@ struct ReleaseOpt {
...
-    #[structopt(
-        possible_values(&version::BumpLevel::variants()),
-        case_insensitive(true),
-        default_value = "release"
-    )]
-    level: version::BumpLevel,
+    #[structopt(case_insensitive(true), default_value = "release")]
```
Comment (epage) [inline src/main.rs]: I believe we could set this up to be an `enum` of either a String or a Level.  That would push the error into arg parsing which might be clearer.

Also, can we still specify variants?  Is that actually used for validation or only for `--help`?

**Comment by sunng87** [comment]
Thanks @epage for your great suggestion! I will update this later today.

File: src/main.rs (hunk 3 above)
Comment (sunng87) [reply src/main.rs]: The variants is for validation so I have to remove it. I wish clap has support for a default field in an arg_enum but unfortunately it doesn't. I will check its support for enum type.

Comment (epage) [reply src/main.rs]: Oh well.

Comment (sunng87) [reply src/main.rs]: I found enum support for structopt has other meaning, which is typically as subcommand. I will leave this as it is for now.

**Pull request merged**